<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Rahul-Predator" failures="4" tests="4" name="Test" time="4.767" errors="0" timestamp="2023-08-13T02:40:54 IST">
  <testcase classname="APIChaining.CreateUser" name="createUser" time="4.588">
    <failure type="java.lang.NumberFormatException" message="For input string: &amp;quot;[null]&amp;quot;">
      <![CDATA[java.lang.NumberFormatException: For input string: "[null]"
at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
at java.base/java.lang.Integer.parseInt(Integer.java:668)
at java.base/java.lang.Integer.parseInt(Integer.java:786)
at java_lang_Integer$parseInt.call(Unknown Source)
at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:45)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:139)
at io.restassured.internal.common.path.ObjectConverter.convertObjectTo(ObjectConverter.groovy:29)
at io.restassured.path.json.JsonPath.getInt(JsonPath.java:245)
at APIChaining.CreateUser.createUser(CreateUser.java:38)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- createUser -->
  <testcase classname="APIChaining.UpdateUser" name="updateUser" time="0.083">
    <failure type="java.lang.NullPointerException" message="Cannot invoke &amp;quot;java.lang.Integer.intValue()&amp;quot; because the return value of &amp;quot;org.testng.ITestContext.getAttribute(String)&amp;quot; is null">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "java.lang.Integer.intValue()" because the return value of "org.testng.ITestContext.getAttribute(String)" is null
at APIChaining.UpdateUser.updateUser(UpdateUser.java:32)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- updateUser -->
  <testcase classname="APIChaining.GetUser" name="testGetUser" time="0.002">
    <failure type="java.lang.NullPointerException" message="Cannot invoke &amp;quot;java.lang.Integer.intValue()&amp;quot; because the return value of &amp;quot;org.testng.ITestContext.getAttribute(String)&amp;quot; is null">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "java.lang.Integer.intValue()" because the return value of "org.testng.ITestContext.getAttribute(String)" is null
at APIChaining.GetUser.testGetUser(GetUser.java:15)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- testGetUser -->
  <testcase classname="APIChaining.DeleteUser" name="DeleteUser" time="0.002">
    <failure type="java.lang.NullPointerException" message="Cannot invoke &amp;quot;java.lang.Integer.intValue()&amp;quot; because the return value of &amp;quot;org.testng.ITestContext.getAttribute(String)&amp;quot; is null">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "java.lang.Integer.intValue()" because the return value of "org.testng.ITestContext.getAttribute(String)" is null
at APIChaining.DeleteUser.DeleteUser(DeleteUser.java:17)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- DeleteUser -->
</testsuite> <!-- Test -->
